thinkhazard/static/less/report.less

```diff
@@ -149,6 +149,26 @@ body {
   top: 10px;
   left: 10px;
   z-index: 2;
+
+  height: 42px;
+  margin: 5px;
```

Comment:
Margin in the document seem to be actually the position of the element regarding the top left corner of the map.
So we have 2 options here:
 - left: 0; top: 0; margin: 5px;
 - left: 5px; top: 5px; (no margin)